### PR TITLE
Adds codification

### PR DESCRIPTION
**Pull request by NejcZdovc**

### CODE REVIEW

File: app/components/footer/index.jsx [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import { useEffect } from 'react'
-import { useLocation, Link } from 'react-router-dom'
-import SlackIcon from '../graphics/icons/slack-icon'
-import TwitterIcon from '../graphics/icons/twitter-icon'
-import LinkedInIcon from '../graphics/icons/linkedin-icon'
-
-export const Footer = () => {
```
Comment (NejcZdovc) [inline app/components/footer/index.jsx:7]: removed as we already have it in header, duplicated code

File: app/components/header/utils.jsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
+import MenuIconDark from '../../graphics/icons/menu-icon-dark'
```
Comment (NejcZdovc) [inline app/components/header/utils.jsx:1]: moved this to utils, so that component is easier to read

File: app/root.jsx [hunk 3]
```diff
@@ -28,45 +28,6 @@ export function links() {
   ]
 }
 
-const addListeners = () => {
```
Comment (NejcZdovc) [inline app/root.jsx:31]: old codification that is not working with the new website

File: app/root.jsx [hunk 4]
```diff
@@ -96,11 +57,8 @@ export default function App() {
       endpointUrl: 'https://sales.blotout.io/sdk',
       customDomain: 'blotout.io',
     })
-    addListeners()
+    capture('Page Navigation', { link: window.location.pathname })
   }, [])
-
-  let data = useRouteData()
```
Comment (NejcZdovc) [inline app/root.jsx:102]: not used

File: app/routes/integrations.jsx [hunk 5]
```diff
@@ -45,25 +50,33 @@ export default function Integrations() {
...
+                    codifyClick('Integration - Box', { type: integration.name })
+                  }
+                  target='_blank'
+                  rel='noreferrer'
+                >
                   <div className='integration-icon'>
                     <img src={integration.icon_url} />
                   </div>
                   <div className='integration-text'>{integration.name}</div>
-                  <a href={integration.git_url} className='integration-link'>
```
Comment (NejcZdovc) [inline app/routes/integrations.jsx:63]: removed as link was moved to the whole box, to simplify click

File: app/routes/integrations.jsx [hunk 6]
```diff
@@ -45,25 +50,33 @@ export default function Integrations() {
...
-                <div className='integration' key={integration.git_url}>
+                <a
+                  href={integration.git_url}
+                  className='integration'
+                  key={integration.git_url}
+                  onClick={() =>
+                    codifyClick('Integration - Box', { type: integration.name })
+                  }
+                  target='_blank'
+                  rel='noreferrer'
```
Comment (NejcZdovc) [inline app/routes/integrations.jsx:73]: Added so user does not leave our site

File: scss/integrations.scss [hunk 7]
```diff
@@ -144,16 +145,6 @@
               margin-top: 30px;
             }
           }
-
-          .integration-link {
```
Comment (NejcZdovc) [inline scss/integrations.scss:148]: not used anymore

**Review by yashghelani — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**